modules/openshift-developer-cli-mirroring-a-supported-builder-image.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster
+
+
+
+.Procedure
+
+= Mirroring a supported builder image
+
+. Verify that the required images tag is not imported:
++
+----
+$ oc describe is nodejs -n openshift
+----
+
+. Mirror the supported image tag to the private registry:
++
+----
+$ oc image mirror registry.access.redhat.com/rhscl/nodejs-10-rhel7:<tag> <private_registry>/rhscl/nodejs-10-rhel7:<tag>
+----
+
+. Import the image:
++
+----
+$ oc tag <mirror-registry>:<port>/rhscl/nodejs-10-rhel7:<tag> nodejs-10-rhel7:latest --scheduled <1>
+----
++
+<1> You must periodically re-import the image. The `--scheduled` flag enables automatic re-import of the image.
```

Comment:
I think you can simply add this information as a sentence and remove the callout. IMHO, If it was a huge block of code and user would be confused about what you are referring too then we could use call outs.